src/api/docker-stats.ts

```diff
@@ -1,60 +1,69 @@
 import { Observable, Observer } from 'rxjs';
 import * as utils from './utils';
 import * as dockerode from 'dockerode';
```

Comment:
remove `dockerode` from that file. should be included only in `docker.ts`

Reply:
done